diff --git a/jquery_update.module b/jquery_update.module
index 956c9e5..135b929 100644
--- a/jquery_update.module
+++ b/jquery_update.module
@@ -75,6 +75,8 @@ function jquery_update_library() {
  * Implements hook_library_alter().
  */
 function jquery_update_library_alter(&$javascript, $module) {
+  global $theme;
+
   $path = drupal_get_path('module', 'jquery_update');
   $version = variable_get('jquery_update_jquery_version', '1.10');
 
@@ -89,7 +91,7 @@ function jquery_update_library_alter(&$javascript, $module) {
     // Replace jQuery with the alternative version.
     $admin_version = variable_get('jquery_update_jquery_admin_version', '');
 
-    if (!empty($admin_version) && path_is_admin(current_path())) {
+    if (user_access('view the administration theme') && !empty($admin_version) && (path_is_admin(current_path()) || $theme == variable_get('admin_theme', 0))) {
       if (version_compare($version, $admin_version, '!=')) {
         $version = $admin_version;
       }
